README.md

```diff
@@ -1,2 +1,75 @@
 # Lykke.Bil2.Templates
-BIL v2 integration services templates
+
+`dotnet new` CLI templates for blockchain integration services - sign service, transaction executor and block reader.
+
+## Templates included
+
+- `lkebil2sign`
+- `lkebil2tx`
+- `lkebil2blocks`
+
+## How to use
+
+1. Clone repository to a local folder;
+2. Install templates. For each internal directory run:
+
+    ```bash
+    $ dotnet new -i ${path}
+    ```
+
+    where `${path}` is the __full__ path to the template directory (where folder .template.config placed) __without trailing slash__.
+
+    ### Example
+
+    To install __blocks reader__ template run
+
+    ```bash
+    dotnet new -i ./Lykke.Bil2.Templates/Lykke.Bil2.BlocksReader
+    ```
+
+3. Generate solutions:
+
+    Generally, run
+
+    ```bash
+    dotnet new ${templateName} -n ${integrationName} -o ${outputPath}
+    ```
+
+    to generate solution in specified directory.
+
+    Or create directory with required name, `cd` into created directory and run
+
+    ```bash
+    dotnet new ${templateName} -n ${integrationName}
+    ```
+
+    to generate solution from the inside.
+
+## Examples
+
+### Sign service
+
+```bash
+dotnet new lkebil2sign -n Bitcoin -o Lykke.Bil2.Bitcoin.SignService
+```
+
```

Comment:
Let's add links to particular readme.md of each service

Reply:
Added links